qiita_db/handlers/prep_template.py

```diff
@@ -63,3 +66,16 @@ def get(self, prep_id):
             response = {'data': pt.to_dataframe().to_dict(orient='index')}
 
         self.write(response)
+
+
+class PrepTemplateAPItestHandler(OauthBaseHandler):
+    @authenticate_oauth
+    def post(self):
+        prep_info_dict = loads(self.get_argument('prep_info'))
```

Comment:
I think it's worth testing with the largest study in our current version of Qiita before deploying because if it breaks we need to rethink this section. Now, this can happen before merging this or after as this is going to its own branch. If you decide we should merge, could you add a point to the list so we make sure it's done and tested before this branch is merged to master?